deploy/docker/cp-api-preconfigure/preconfigure-api

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+# Copyright 2022 EPAM Systems, Inc. (https://www.epam.com/)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function is_dir_empty {
+    local dir_path="$1"
+
+    if [ ! -d "$dir_path" ]; then
+        mkdir -p "$dir_path"
+        return 0
+    fi
+
+    shopt -s nullglob
+    shopt -s dotglob
+
+    local dir_contents=(${1}/*)
+
+    shopt -u nullglob
+    shopt -u dotglob
+
+    local result=0
+    (( ${#dir_contents[*]} )) && result=1 || result=0
+    return $result
+}
+
+function configure_icon_win_distribution() {
+    # Check and change windows icon if exists
+    CLOUD_DATA_WIN_ICO_NAME=${CLOUD_DATA_WIN_ICO_NAME:-"favicon.ico"}
+    if [ "$CP_PREF_API_STATIC_DIRECTORY_EXT" ] && ! is_dir_empty "$CP_PREF_API_STATIC_DIRECTORY_EXT"; then
+        if [ "$CP_PREF_API_STATIC_DIRECTORY_EXT/$CLOUD_DATA_WIN_ICO_NAME" ]; then
+            xvfb-run -f $JENKINS_HOME/.Xauthority -a wine $JENKINS_HOME/reshack/ResourceHacker.exe -open $CP_COMMON/cloud-data-win32-x64/cloud-data.exe -save $CP_COMMON/cloud-data-win32-x64/cloud-data.exe -action addoverwrite -res $CP_PREF_API_STATIC_DIRECTORY_EXT/favicon.ico -mask ICONGROUP,MAINICON,
```

Comment:
Seems, like
> ... -res $CP_PREF_API_STATIC_DIRECTORY_EXT/favicon.ico  ..

Should be changed to:
> ... -res $CP_PREF_API_STATIC_DIRECTORY_EXT/$CLOUD_DATA_WIN_ICO_NAME ...

in case the icon name is overridden.

It could be extracted to some variable

> CLOUD_DATA_WIN_ICO_PATH="$CP_PREF_API_STATIC_DIRECTORY_EXT/$CLOUD_DATA_WIN_ICO_NAME"

to be re-used in `if` condition and both `xvfb-run` calls

Reply:
Done